pkg/stat/set.go

```diff
@@ -71,10 +72,12 @@ type graph struct {
 }
 
 type line struct {
-	desc string
-	rate bool
-	data []float64
-	hist []*gohistogram.NumericHistogram
+	name  string
```

Comment:
Are different names possible? If not - you can use them as a sorting key.

Reply:
Names are unique, but the creation order is inentional:

> Sort lines based on creation order.
> This provides stable order and gives a user some control over order.